src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -284,12 +284,13 @@ export default function ChartContainer<T extends ChartDataTypes>({
     }
   }, [highlightedX, showPopover]);
 
-  const onSVGMouseDown = () => {
+  const onSVGMouseDown = (e: React.MouseEvent<SVGSVGElement, MouseEvent>) => {
     if (isPopoverOpen) {
       if (isPopoverPinned) {
         dismissPopover();
       } else {
         pinPopover();
+        e.preventDefault();
```

Comment:
why prevent default is needed?

Reply:
Otherwise the click event continues, causing focus to be moved to the clicked element (the chart itself)

Reply:
Do we need to apply similar change to area-chart and pie-chart then?

Reply:
yes actually, seems these weren't covered by any tests, added the change & tests for these two as well